electron/app/initKubeconfig.ts

```diff
@@ -61,6 +61,7 @@ export const getKubeConfigContext = (configPath: string): KubeConfig => {
     return {
       path: configPath,
       isPathValid: false,
+      contexts: [],
```

Comment:
This should actually fix all the problem - @WitoDelnat Could you please test?

Reply:
Unfortunate I can't. I have no way to reproduce what I had. Simply ran into it once and noticed it was reported as well.